init.js

```diff
@@ -2,8 +2,16 @@
 
 const path = require('path')
 const Module = require('module')
+const telemetry = require('./packages/dd-trace/src/telemetry/init-telemetry')
+const Config = require('./packages/dd-trace/src/config')
+const log = require('./packages/dd-trace/src/log')
+
+// eslint-disable-next-line no-new
+new Config() // we need this to initialize the logger
 
 let initBailout = false
+let clobberBailout = false
+const forced = ['1', 'true', 'True'].includes(process.env.DD_INJECT_FORCE)
```

Comment:
I often see this `['1', 'true', 'True'].includes(…)` idiom. We do have an `isTrue` function in `util.js` though – could you use that instead for consistency? It also accepts the word "true" in any combination of letter cases and while "tRuE" is arguably weird, "TRUE" is reasonable. 
```suggestion
const forced = isTrue(process.env.DD_INJECT_FORCE)
```
Even better, since you're creating a `Config` object on line 10 anyway you could let it read the env variable and expose its evaluation result as a property of the config object. That way the config object is the only source of configuration for all of the rest of the codebase. There's already some processing in there for e.g. `DD_INJECTION_ENABLED` that I added for profiler's needs, but you'd be welcome to generalize it, and then you could use a Config property for that as well.